Update ansys.md about unlimited license (#735)
NREL/HPC: docs/Documentation/Applications/comsol.md

## Running a Single-Node COMSOL Model in Batch Mode
You can save your model built in FastX+GUI mode into a file such as `myinputfile.mph`. Once that's available, the following job script shows how to run a single process multithreaded job in batch mode:

> ???+ example "Example Submission Script"

Reviewer: These headers should be left in. They render as the drop down boxes in the site

(https://squidfunk.github.io/mkdocs-material/reference/admonitions/)

Author: Added in my repo.

## Running a Multi-Node COMSOL Model in Batch Mode
To configure a COMSOL job with multiple MPI ranks, required for any job where the number of nodes >1, you can build on the following template:

> ???+ example "Example Multiprocess Submission Script"

Reviewer: These headers should be left in. They render as the drop down boxes in the site

(https://squidfunk.github.io/mkdocs-material/reference/admonitions/)

Author: added in my repo.

The job script can be submitted to SLURM just the same as above for the single-node example. The option `-mpibootstrap slurm` helps COMSOL to deduce runtime parameters such as `-nn`, `-nnhost` and `-np`. For large jobs that require more than one node, this approach, which uses MPI and/or OpenMP, can be used to efficiently utilize the available resources. Note that in this case, we choose 32 MPI ranks, 8 per node, and each rank using 13 threads for demonstration purpose, but *not* as an optimal performance recommendation. The optimal configuration depends on your particular problem, workload, and choice of solver, so some experimentation may be required.

> ## Running COMSOL Model with GPU

Reviewer: ```suggestion
## Running a COMSOL Model with GPU
```

In COMSOL Multiphysics®, GPU acceleration can significantly increase performance for time-dependent simulations that use the discontinuous Galerkin (dG) method, such as those using the Pressure Acoustics, Time Explicit interface, and for training deep neural network (DNN) surrogate models. The following is a job script example used to run COMSOL jobs on GPU nodes.

> 

Reviewer: ```suggestion
???+ example "Example GPU Job Script"
```

Author: added in my repo.

     ```
     #!/bin/bash
     #SBATCH --job-name=comsol-batch-GPUs
     #SBATCH --time=00:20:00
     #SBATCH --gres=gpu:1  # request 1 gpu per node, each gpu has 80 Gb of memory
     #SBATCH --mem-per-cpu=2G # requested memory per CPU core

>      #SBATCH --ntasks-per-node=64

Reviewer: Is this many cores necessary? This will then charge for half a gpu node even though the job is only using 1 GPU.

Author: Changed it to 20.

     #SBATCH --nodes=2
     #SBATCH --account=<allocation handle> 
     #SBATCH --output=comsol-%j.out
     #SBATCH --error=comsol-%j.err
     
     # This helps ensure your job runs from the directory
     # from which you ran the sbatch command
     cd $SLURM_SUBMIT_DIR
     
     # Set up environment, and list to stdout for verification
     module load comsol
     echo " "
     module list
     echo " "
     
     inputfile=$SLURM_SUBMIT_DIR/myinputfile.mph
     outputfile=$SLURM_SUBMIT_DIR/myoutputfilename
     logfile=$SLURM_SUBMIT_DIR/mylogfilename
     
     # Run a 2-node, 64-rank parallel COMSOL job with 1 threads for each rank and 1 gpu per node
     # -nn = total number of MPI ranks
     # -nnhost = number of MPI ranks per host
     # -np = number of threads per rank
     
     comsol –nn 128 -nnhost 64 batch -np 1 -inputfile $inputfile -outputfile $outputfile –batchlog $logfile
     ```

> Note, When launching a GPU job on Kestrel, be sure to do so from one of its dedicated GPU login nodes (ssh to Kestrel from the NREL network using kestrel-gpu.hpc.nrel.gov).

Reviewer: ```suggestion
Note, when launching a GPU job on Kestrel, be sure to do so from one of its dedicated [GPU login nodes](../Systems/Kestrel/index.md).
```
